cla_frontend/apps/zendesk/client.py

```diff
@@ -0,0 +1,81 @@
+import json
+import requests
+from requests.exceptions import ConnectionError, Timeout
+from django.conf import settings
+from django.template import loader, Context
+
+TICKETS_URL = 'https://ministryofjustice.zendesk.com/api/v2/tickets.json'
+ZENDESK_REQUESTER_ID = 762871298 # Civil Legal Advice
+ZENDESK_GROUP_ID = 24287107 # CLA Operator/Provider
+REQUEST_TIMEOUT = 10
+
+
+class ZendeskResponse(object):
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
```

Comment:
From the way it's used later, this should be:
```python
class ZendeskResponse(object):

    def __init__(self, json, status):
        self.json = json
        self.status = status
```
But then it's just a data object with no methods, so would probably be better off as a dict.